src/box/vy_cache.c

```diff
@@ -226,9 +226,11 @@ vy_cache_env_set_quota(struct vy_cache_env *env, size_t quota)
 
 void
 vy_cache_add(struct vy_cache *cache, struct vy_entry curr,
-	     struct vy_entry prev, struct vy_entry key,
-	     enum iterator_type order)
+	     struct vy_entry prev, bool is_boundary,
```

Comment:
Nit: `is_boundary` is a stale name - the flag no longer indicates whether the tuple is the final tuple in the chain - only the starting one.

Reply:
Fair enough. Renamed `vy_cache_add::is_boundary` and `vy_read_iterator::is_cache_boundary` to `vy_cache_add::is_first` and `vy_read_iterator::is_first_cached`, respectively